shared/lib/entity-mocks.ts

```diff
@@ -23,6 +24,14 @@ export const createUser = async (
   return { ...user, password: usedPassword } as User;
 };
 
+export const createQuestionIndicatorMap = async (
+  dataSource: DataSource,
+  additionalData: QuestionIndicatorMap,
```

Comment:
If the parameter is named `additionalData`, it should accept something **additional**.

Please add logic so that this mock utility can generate something that is useful without the need to pass additional values, take into consideration unique constraints as well